.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report that describes an issue
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Please replace the text in the individual sections below. -->
+
+### Description
+
+A concise description of what is causing an issue and what you expected to happen.
+
+### To reproduce
+
+Please copy & paste the code that is triggering the issue into this section and describe the context and prior steps that lead to the problem as detailed as possible.
+
+### Error output
```

Comment:
It might make more sense to put "Error output" above "To reproduce."

Reply:
Hmm.. not sure. The user would first try to create a minimal example to reproduce the issue and then copy & paste the output. I think it makes more sense the way it is.